src/test/java/com/exasol/adapter/request/LoggingConfigurationTest.java

```diff
@@ -64,4 +67,10 @@ void testFallBackToLocalLoggingInCaseOfIllegalLoggingPort() {
         final LoggingConfiguration configuration = createLoggingConfiguration(this.properties);
         assertThat(configuration.isRemoteLoggingConfigured(), equalTo(false));
     }
-}
\ No newline at end of file
+
+    @Test
+    void testSerializable() throws ClassNotFoundException, IOException {
+        final LoggingConfiguration config = createLoggingConfiguration(this.properties);
+        SerializationTestUtil.assertSerializable(config);
```

Comment:
Shouldn't we somehow verify that the field values are identical?

Reply:
Yes, we should ;)